documentation/src/main/doc/modules/connectors/partials/META-INF/connector/smallrye-kafka-incoming.adoc

```diff
@@ -37,6 +37,10 @@ Type: _boolean_ | false | `true`
 
 Type: _boolean_ | false | `true`
 
+| *kafka-configuration-name* | The name set in `javax.inject.Named` of a bean that provides the default Kafka consumer/producer (`Map<String, Object>`) configuration to use for this channel. The channel configuration can still override any attribute.
```

Comment:
So we can use for this name of the service binding that will somehow work or actually do provide extra field that will be uniquely naming the service:

I have proposed something to the spec (but we can do it ourselves)
https://github.com/k8s-service-bindings/spec/issues/187

We can do it for kafka and any future services.

[Note] this is just though - binding name is ok for now